### PR TITLE
add OWNERS for EAP XP3 Helm chart

### PR DESCRIPTION
**Pull request by jmesnil**
Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

### CODE REVIEW

**Comment by jmesnil** [comment]
@dperaza4dustbit could you please review?

File: charts/redhat/redhat/eap-xp3/OWNERS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+chart:
+  name: eap-xp3
+  shortDescription: Build and Deploy EAP XP 3 applications on OpenShift
+publicPgpKey: null
+users:
+  - githubUsername: bstansberry
+  - githubUsername: jmesnil
+vendor:
+  label: redhat
+  name: Red Hat JBoss Entreprise Application Platform expansion pack
```
Comment (dperaza4dustbit) [inline charts/redhat/redhat/eap-xp3/OWNERS:10]: This looks to be more of a description of the chart right? vendor.name is a UI friendly name for the vendor in this case Red Hat right? Can you change for your other owners file as well? I merged that one before realizing this.

**Review by dperaza4dustbit — CHANGES_REQUESTED** [review]

File: charts/redhat/redhat/eap-xp3/OWNERS (hunk 1 above)
Comment (jmesnil) [reply charts/redhat/redhat/eap-xp3/OWNERS:10]: @dperaza4dustbit I've no issue changing this field but I looked at the other charts and they all use the `name` to specify the products:

https://github.com/openshift-helm-charts/charts/blob/95963250806f7a7c515d539767dfbfee6adbad47/charts/redhat/redhat/datagrid/OWNERS#L9

https://github.com/openshift-helm-charts/charts/blob/95963250806f7a7c515d539767dfbfee6adbad47/charts/redhat/redhat/quarkus/OWNERS#L9

Comment (jmesnil) [reply charts/redhat/redhat/eap-xp3/OWNERS:10]: @dperaza4dustbit I've updated the OWNERS file in the same PR that submitted the eap74 chart in #319

Comment (jmesnil) [reply charts/redhat/redhat/eap-xp3/OWNERS]: I've amended the commit for this one (eap-xp3)

**Review by dperaza4dustbit — APPROVED** [review]
lgtm, thanks, made changes to others on PR 320

**Pull request merged**